tsconfig.json

```diff
@@ -12,9 +12,9 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "noImplicitAny": false,
```

Comment:
I was getting a Typescript error when running `yarn start` so I turned off this rule as a workaround. Possible there's a newer version of one of the packages that wouldn't hit this problem.